modules/fts/pages/fts-sorting.adoc

```diff
@@ -222,6 +227,7 @@ When you sort results on a field that is not indexed, or when a particular docum
 
 The same characters may render differently when using a graphic tool or command line tools like `jq`.
 
+[source,json]
 ----
       "sort": [
         "����������",
```

Comment:
What is this?

Reply:
This is correct as per the topic description. These are a series of Unicode non-printable characters that appear in the sort field: